src/index.js

```diff
@@ -8,4 +9,7 @@ else {
     const path = window.location.pathname
     const strippedPath = path.substring(1,path.length-1);
     attemptToRenderTable(strippedPath);
+    startVideo()
```

Comment:
the attemptToRenderTable function doesn't always bring someone to a room. If they aren't authenticated, it will bring them to the home page, in which case we wouldn't want video to start. So, I think the startVideo function should actually be right after "await renderTable(tableNumber)" on line 17 of the attemptToRenderTable function in src/views/table.js

Reply:
It may actually make more sense to put it in the renderTable function itself